tools/parity_scale_decoder/src/main.rs

```diff
@@ -265,9 +265,10 @@ mod tests {
             Repeats::Indefinitely,
             account_id,
             // FIXME: rewrite the filters using the builder DSL https://github.com/hyperledger/iroha/issues/3068
```

Comment:
While I would not consider what I have done as an event filters DSL, it does greatly simplify creation of event filters.

The question would be: do we still want an event filters DSL or is the current way to define event filters sufficient? I imagine it looking somewhat like this:

```rust
EventFilterBox::account().with_id(account_id);
EventFilterBox::account().only(AccountEventMatcher::ByCreated);
EventFilterBox::account().only(AccountEventMatcher::ByCreated).with_id(account_id);
```

It would make things nicer when [id matchers](https://github.com/hyperledger/iroha/pull/4240#issuecomment-1920697485) are implemented, because `with_id` can be made to accept `Into<AccountIdMatcher>`, but otherwise the usability benefit seems marginal

Reply:
You can remove, we have a ticket that tracks it

Reply:
You can remove, we have a ticket that tracks it